src/sentry/models/group.py

```diff
@@ -803,6 +804,23 @@ def get_recommended_event_for_environments(
             else self.get_latest_event_for_environments([env.name for env in environments])
         )
 
+    def get_suspect_commit(self) -> Commit | None:
+        from sentry.models.groupowner import GroupOwner, GroupOwnerType
+
+        suspect_commit_owner = GroupOwner.objects.filter(
```

Comment:
Looking [here](https://github.com/getsentry/sentry/blob/a297c789d9c2dbf99d74fdb1872eeb078281b268/src/sentry/utils/committers.py#L305) we order by date_added and filter out groupowner rows without context, may want to do that here as well just to ensure we're getting the same one.